test/unit/spec/nbextensions/bulkImportCell/tabs/configure-spec.js

```diff
@@ -51,6 +51,7 @@ define([
 
         beforeEach(() => {
             runtime = Runtime.make();
+            bus = runtime.bus();
```

Comment:
moved this line up whilst looking at these tests for "inspiration" for the XSV tests